docs/Demos.md

```diff
@@ -7,6 +7,196 @@ title: "React-admin demos"
 
 If you want to see what react-admin is capable of, or if you want to learn from apps built by seasoned react-admin developers, check out these demos.
 
+## Overview
+
+Discover our demos that show you the react-admin potential and help you familiarize with it.
+
+<style>
+    .demos-list {
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+        grid-template-rows: repeat(3, 1fr);
+        grid-gap: 10px;
+    }
+
+    @media (max-width: 600px) {
+        .demos-list {
+            display: flex;
+            flex-wrap: wrap;
+            gap: 10px;
+        }
+    }
+
+    .card {
+        box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+        transition: 0.3s;
+        background-color: rgba(243, 246, 249, 0.4);
+        border: 1px solid rgba(0,0,0,0.25);
+        border-radius: 30px;
+        margin: 0;
+        display: flex;
+        flex-direction: column;
+    }
+
+    .card:hover {
+        box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+    }
+
+    .card > img, .card > video {
+        border-radius: 15px;
+        width: 100%;
```

Comment:
Videos are not centered, there is more right margin because of the `.markdown-section img, .markdown-section video` rules.
![image](https://github.com/marmelab/react-admin/assets/7949510/7fa04efb-1a67-4d62-875c-0a9de2121da2)
This code will fix this:
```suggestion
        max-width: 100%;
```